### ISSUE TITLE
java.lang.UnsatisfiedLinkError when using libstt-1.0.0-*.aar

### ISSUE THREAD

**Issue by comodoro** [coqui-ai/STT#1997]
I could not get the aar work on Android with custom code and models, so I tried the [English release model](https://coqui.ai/english/coqui/v1.0.0-huge-vocab) and the [Android STTDemo app](https://github.com/coqui-ai/STT-examples/tree/r1.0/android_mic_streaming) adapted for `libstt-1.0.0-release.aar`. I got java.lang.UnsatisfiedLinkError not finding either `libkenlm.so` or `libstt-jni.so` with an emulator as well as two devices. Also tried `libstt-0.10-alpha.28`, same result.

Steps to reproduce:
Download the libstt aar and use it in the STTDemo app in Android Studio, either in `libs` or a local Maven repository. After pressing `Start recording`, `java.lang.UnsatisfiedLinkError` occurs:

```
2021-10-24 11:04:20.671 480-480/? D/VM_BMS: wake_lock released
2021-10-24 11:04:22.565 21858-21858/? W/linker: /data/app/org.sttdemo-1/lib/arm/libstt.so: unused DT entry: type 0x1d arg 0x8d4e4
2021-10-24 11:04:22.571 21858-21858/? D/AndroidRuntime: Shutting down VM
    
    
    --------- beginning of crash
2021-10-24 11:04:22.587 21858-21858/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.sttdemo, PID: 21858
    java.lang.IllegalStateException: Could not execute method for android:onClick
        at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:390)
        at android.view.View.performClick(View.java:5205)
        at android.view.View$PerformClick.run(View.java:21164)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5484)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:385)
        at android.view.View.performClick(View.java:5205) 
        at android.view.View$PerformClick.run(View.java:21164) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5484) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
     Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: library "libkenlm.so" not found
        at java.lang.Runtime.loadLibrary(Runtime.java:372)
        at java.lang.System.loadLibrary(System.java:1076)
        at ai.coqui.libstt.STTModel.<clinit>(STTModel.java:9)
        at org.sttdemo.MainActivity.createModel(MainActivity.kt:89)
        at org.sttdemo.MainActivity.onRecordClick(MainActivity.kt:119)
        at java.lang.reflect.Method.invoke(Native Method) 
        at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:385) 
        at android.view.View.performClick(View.java:5205) 
        at android.view.View$PerformClick.run(View.java:21164) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5484) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
2021-10-24 11:04:22.621 1153-2634/? D/ActivityManager: Prepare to send crash information Messege to SystemGaService
2021-10-24 11:04:22.623 1153-1179/? D/SystemGaService: sendCrashInfoToRemote successful
2021-10-24 11:04:22.692 1153-2634/? W/ActivityManager:   Force finishing activity org.sttdemo/.MainActivity
```
or:

```
2021-10-25 09:29:54.828 3218-3218/? D/AndroidRuntime: Shutting down VM
    
    
    --------- beginning of crash
2021-10-25 09:29:54.835 3218-3218/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.sttdemo, PID: 3218
    java.lang.IllegalStateException: Could not execute method for android:onClick
        at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:390)
        at android.view.View.performClick(View.java:7448)
        at android.view.View.performClickInternal(View.java:7425)
        at android.view.View.access$3600(View.java:810)
        at android.view.View$PerformClick.run(View.java:28305)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:385)
        at android.view.View.performClick(View.java:7448) 
        at android.view.View.performClickInternal(View.java:7425) 
        at android.view.View.access$3600(View.java:810) 
        at android.view.View$PerformClick.run(View.java:28305) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7656) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
     Caused by: java.lang.UnsatisfiedLinkError: dlopen failed: library "libstt-jni.so" not found
        at java.lang.Runtime.loadLibrary0(Runtime.java:1087)
        at java.lang.Runtime.loadLibrary0(Runtime.java:1008)
        at java.lang.System.loadLibrary(System.java:1664)
        at ai.coqui.libstt.STTModel.<clinit>(STTModel.java:9)
        at org.sttdemo.MainActivity.createModel(MainActivity.kt:89)
        at org.sttdemo.MainActivity.onRecordClick(MainActivity.kt:119)
        at java.lang.reflect.Method.invoke(Native Method) 
        at androidx.appcompat.app.AppCompatViewInflater$DeclaredOnClickListener.onClick(AppCompatViewInflater.java:385) 
        at android.view.View.performClick(View.java:7448) 
        at android.view.View.performClickInternal(View.java:7425) 
        at android.view.View.access$3600(View.java:810) 
        at android.view.View$PerformClick.run(View.java:28305) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7656) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
2021-10-25 09:29:54.872 288-3057/? W/android.hardware.audio.service.ranchu: TinyalsaSink::write:125 pcm_write was late reading frames, dropping 
```

**Comment by comodoro** [comment]
> One important debugging trick for transcription problems is to dump PCM samples to a file immediately before you feed them to STT, so you can listen to exactly what's being input into the model by yourself. Often on mobile devices there are issues with dropped frames when transcription is done in the same thread as recording, for example.

That's a great advice, thanks! There are indeed frames missing as well, but it is the `sttdemo` app that does the transcription in a thread, so I don't know how to fix it. Maybe increase buffer size?

**Comment by reuben** [comment]
Fixed in 1.1.0
